service/service.go

```diff
@@ -91,9 +90,27 @@ func New(config Config) (*Service, error) {
 		return nil, microerror.Mask(err)
 	}
 
-	var appController *controller.App
+	var appController *app.App
+	{
+		c := app.Config{
+			G8sClient:    g8sClient,
+			Logger:       config.Logger,
+			K8sClient:    k8sClient,
+			K8sExtClient: k8sExtClient,
+
+			ProjectName:    config.ProjectName,
+			WatchNamespace: config.Viper.GetString(config.Flag.Service.Kubernetes.Watch.Namespace),
+		}
+
+		appController, err = app.NewApp(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var appCatalogController *appcatalog.AppCatalog
```

Comment:
Lets make this controller first. Generally we always try and order code alphabetically so its cleaner easier to navigate. Also in this case we boot the appCatalog controller first.